content/guides/dashboard/organizations.md

```diff
@@ -78,24 +78,27 @@ to another organization before you can delete the organization.
 
 ### Open Source Plan
 
-To support the community, we provide the Open Source (OSS) plan for public
-projects to take advantage of our Dashboard Service with unlimited test runs. To
-qualify, your project needs two things:
+To support the community, we provide the Open Source (OSS) plan to take
+advantage of our Dashboard Service with unlimited test runs. To qualify, your
+Organization needs the following:
 
-- Your project is a non-commercial entity
-- Source code for your project is available in a public location with an
+- Project is a non-commercial entity
```

Comment:
The points are clear, but the new text reads a little awkwardly. Maybe something like:

- A project that is a non-commercial entity
- A project that is public
- Your project source code is available in a public location with an OSI-approved license
- _Hit the Free Plan limit of 500 Test Recordings_

I'm not sure how to tweak the last one because I'm not sure what we're getting at here. Is the point that they can't exceed that limit of 500 tests?

Reply:
Cool. That feels better. I'm still fuzzy on the last bullet though and this is more for my own edification...

A project qualifies for the OSS tier, if it hits a limit of test 500 runs? Or is it more like the project exists within/under the Free plan limit of 500 runs in order to be on this plan? It may just be the use of _hit_ that seems confusing.

Reply:
Another great point. I changed it to reach. The goal is to see some significant recording activity in the Dashboard.